tt_eager/tt_dnn/op_library/sharded/multi_core/sharded_op_multi_core.cpp

```diff
@@ -379,6 +390,18 @@ operation::ProgramWithCallbacks sharded_to_interleaved_multi_core(const Tensor&
     tt_metal::SetRuntimeArgs(program, unary_reader_kernel_id, all_cores, {num_units_per_shard});
 
     uint32_t curr_idx_h = 0, curr_idx_w = 0;
+    if (num_slices > 1) {
```

Comment:
Duplicate code. Only difference is tensor (input/output). To me it looks better to have function that will return start_idx_h.

Reply:
Will do